src/test/db/test_GEOmetadb_dataset_linker.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from src.db.GEOmetadb_dataset_linker import GEOmetadbDatasetLinker
+from parameterized import parameterized
+from typing import List
+
+class TestGEOmetadbDatasetLinker(unittest.TestCase):
+    def setUp(self):
+        self.GEOmetadb_dataset_linker = GEOmetadbDatasetLinker(GEOmetadb_path="src/test/db/testGEOmetadb.sqlite")
```

Comment:
Where do I find the instruction how to create testGEOmetadb.sqlite? In case we modify schema for example?
Also please keep filenames lowercase.